### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by rotsen18**

### CODE REVIEW

File: app/game_config.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+"""
+In game config you can set configuration for battle.
+Statistic for every knight is located in KNIGHTS.
+In FIGHTS you can write as many battles as you wish.
+Every battle is element of the list with two persons.
+"""
+
+FIGHTS = [
```
Comment (KatiukhaO) [inline app/game_config.py:8]: name of variable in lower case

**Review by KatiukhaO — CHANGES_REQUESTED** [review]

File: app/game_config.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    ("lancelot", "mordred"),
+    ("arthur", "red_knight")
+]
+
+KNIGHTS = {
```
[the remark this answers is not in the document]
Comment (rotsen18) [reply app/game_config.py:13]: I was guided by it: https://peps.python.org/pep-0008/#constants
What you think?

[the remark this answers is not in the document]
Comment (KatiukhaO) [reply app/game_config.py:13]: in this guided i find, about name variables: https://peps.python.org/pep-0008/#function-and-variable-names

[the remark this answers is not in the document]
Comment (KatiukhaO) [reply app/game_config.py:13]: o, sorry, I understand this is a module level constant, then is OK, that name constant in uppercase

**Review by KatiukhaO — APPROVED** [review]
good work, especially, i like, your realization function battle

**Review by MasterpieceElbow — APPROVED** [review]

**Pull request closed without merging**